src/components/audio-player/audio-player.stories.js

```diff
@@ -0,0 +1,159 @@
+import html from "../../../.storybook/helpers/html";
+import { useEffect } from "@storybook/client-api";
+import { withKnobs, text, boolean } from "@storybook/addon-knobs";
+import initializeAudioPlayers from "./audio-player.js";
+import greekHall1x1 from "../../../.storybook/assets/images/greek-hall/1x1";
+
+import { playIcon, pauseIcon, rewindTenSecondsIcon, forwardTenSecondsIcon, upCarrotIcon } from './helper-icons/helper-icons.js';
+
+export default { title: "Media/Audio Player", decorators: [withKnobs] };
+
+const data = () => {
+ return {
+		darkMode: boolean("Dark Mode", false),
+		track: {
+			hasImage: boolean ("Has Image", true),
+			audioFileURL:
+				text("Audio File URL", "https://cdn.bitdegree.org/learn/I_Cactus_-_05_-_ruby_cactus.mp3?raw=true"),
+			coverImage: greekHall1x1,
+			subtitle: text("Subtitle", "Praise Songs about Javascript"),
+			title: text("Title", "Track 1. Title"),
+			transcript: text(
+				"Transcript",
+				"Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpaqui officia deserunt mollit anim id est laborum."
+			),
+		}
+	};
+};
+
+const audioPlayerMarkUp = (model, playerMode = "") => {
```

Comment:
Do you think, since the audio player requires so much specific markup, and needs JS anyways, that we should maybe just use an approach with a "mount point" that has data, and then the `AudioPlayer` can just render itself into that via a selector? (Kinda like Flickity or other libraries?)

Ie, instead of every downstream project needing to get the html and classes _just so_

It's more like:


```html
<div class="audio-player-mount-point" data-audio-player="{darkMode:true,track:{hasImage:...etc}}"></div>

Reply:
We decided "no," for now!